base/reflection.jl

```diff
@@ -291,11 +291,9 @@ function code_typed(f::ANY, types::ANY=Tuple; optimize=true)
     for x in _methods(f,types,-1)
         linfo = func_for_method_checked(x, types)
         if optimize
-            (tree, ty) = Core.Inference.typeinf(linfo, x[1], x[2], linfo,
-                                                true, true)
+            (tree, ty) = Core.Inference.typeinf(linfo, x[1], x[2], true)
```

Comment:
I recently made the exact same change to the signature of `typeinf` on jb/linear. That's encouraging :)